css/style.css

```diff
@@ -0,0 +1,910 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto&display=swap");
+
+:root {
+  --color-primary: #272a31;
+  --color-secondary: #ec5242;
+  --color-tertiary: #d3d3d3;
+  --color-white: #fff;
+  --color-grey: #585858;
+  --color-light-grey: #ebf0ee;
+}
```

Comment:
The pages use the same exact fonts from the original design 💪 Well done!

Reply:
Thanks